packages/core/contracts/oracle/implementation/VotingV2.sol

```diff
@@ -1141,7 +1147,24 @@ contract VotingV2 is
                 voteInstance.resultComputation.getResolvedPrice(_computeGat(priceRequest.lastVotingRound));
             return (true, resolvedPrice, "");
         } else if (requestStatus == RequestStatus.Future) return (false, 0, "Price is still to be voted on");
-        else return (false, 0, "Price was never requested");
+        else {
+            (bool previouslyResolved, int256 previousPrice) =
+                _getPriceFromPreviousVotingContract(identifier, time, ancillaryData);
+            if (previouslyResolved) return (true, previousPrice, "Returned from previous contract");
+            else (false, 0, "Price was never requested");
```

Comment:
```suggestion
           else return (false, 0, "Price was never requested");
```
I think we are missing a return here. And maybe we could test this last case in the tests.